crates/autopilot/src/domain/auction/mod.rs

```diff
@@ -13,6 +14,7 @@ pub struct Auction {
     pub latest_settlement_block: u64,
     pub orders: Vec<Order>,
     pub prices: BTreeMap<H160, U256>,
+    pub surplus_capturing_jit_order_owners: Vec<H160>,
```

Comment:
I know other fields are not yet converted to domain types but let's use here
`pub surplus_capturing_jit_order_owners: Vec<eth::Address>`

Reply:
@sunce86 when to use one or the other? the line above has `H160`, so I never know when to use `Address` or `H160` :thinking:

Reply:
if inside `domain` we should use domain type - Address.

Yeah, this `Auction` struct doesn't respect that and it should be fixed in a separate PR.